src/Workspaces/Core/Portable/Options/IGlobalOptionService.cs

```diff
@@ -27,12 +27,24 @@ internal interface IGlobalOptionService
         [return: MaybeNull]
         T GetOption<T>(Option<T> option);
 
+        /// <summary>
+        /// Gets the current value of the specific option.
+        /// </summary>
+        [return: MaybeNull]
```

Comment:
📝 I believe this annotation is incorrect starting with #41627. We should update this service (and possibly `IOptionService`) to account for this change.

Reply:
Sure, this can be done as a follow-up change. I will track it in the follow-up issue I filed.

Reply:
Added a follow-up item in https://github.com/dotnet/roslyn/issues/41462